recipes/pylibdmtx/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pylibdmtx" %}
+{% set version = "0.1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: "https://github.com/NaturalHistoryMuseum/pylibdmtx.git"
+  git_rev: "v{{ version }}"
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - Pillow
```

Comment:
Do you need numpy here? Does the package import numpy directly?

Reply:
Removed